common/lib/xmodule/xmodule/js/src/video/09_poster.js

```diff
@@ -0,0 +1,79 @@
+(function (define) {
+'use strict';
+define('video/09_poster.js', ['video/00_resizer.js'], function (Resizer) {
+    /**
+     * Poster module.
+     * @exports video/09_play_skip_control.js
+     * @constructor
+     * @param {Object} element
+     * @param {Object} options
+     */
+    var VideoPoster = function (element, options) {
+        if (!(this instanceof VideoPoster)) {
+            return new VideoPoster(element, options);
+        }
+
+        _.bindAll(this, 'onClick', 'destroy');
+        this.dfd = $.Deferred();
+        this.element = element;
+        this.container = element.find('.video-player');
+        this.options = options || {};
+        this.initialize();
+    };
+
+    VideoPoster.moduleName = 'Poster';
+    VideoPoster.prototype = {
+        template: _.template([
+            '<div class="video-pre-roll is-<%= type %> poster" ',
+                'style="background-image: url(<%= url %>)">',
+                '<span tabindex="0" class="btn-play" aria-label="',
```

Comment:
> I cannot test this feature at all because this line makes it entirely inaccessible. There is a great HTML element <button> that allows you to put images in it (like the triangle here). The image would have an alt attribute (Play Video) which serves as the accessible text. The <button> element comes with all sorts of free properties and behaviors (you can move focus to it, click it, use the enter or spacebar to activate it, it populates the accessibility API with the accessible name, etc.) <span> doesn't do any of this. Unless you want to put all the effort into recreating all that great stuff you get for free with <button>, I would just use <button>. Let's try this:
> 
>  <button><img src="path/to/playicon.png" alt="Play Video" /></button>

Fixed